### DISCUSSION TITLE
Security Onion in Hybrid Env.

### DISCUSSION THREAD

**Discussion by geoleb** [Security-Onion-Solutions/securityonion#6962] [Unsupported Versions] [answered]
Is there a way I can implement security onion on Azure Cloud and on Prem in a single master node?

**Comment by dougburks** [comment, marked as answer]
You could build a distributed deployment with manager and search nodes in Azure and forward nodes on prem:
https://docs.securityonion.net/en/2.3/architecture.html#distributed
https://docs.securityonion.net/en/2.3/cloud-azure.html
